spec-tests/src/test/java/org/candlepin/spec/owners/OwnerResourceConsumerFactFilterSpecTest.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2009 - 2022 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.spec.owners;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.candlepin.dto.api.client.v1.ConsumerDTO;
+import org.candlepin.dto.api.client.v1.ConsumerDTOArrayElement;
+import org.candlepin.dto.api.client.v1.OwnerDTO;
+import org.candlepin.dto.api.client.v1.UserDTO;
+import org.candlepin.invoker.client.ApiException;
+import org.candlepin.spec.bootstrap.client.ApiClient;
+import org.candlepin.spec.bootstrap.client.ApiClients;
+import org.candlepin.spec.bootstrap.client.SpecTest;
+import org.candlepin.spec.bootstrap.data.builder.Consumers;
+import org.candlepin.spec.bootstrap.data.builder.Owners;
+import org.candlepin.spec.bootstrap.data.util.UserUtil;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Map;
+
+@SpecTest
+public class OwnerResourceConsumerFactFilterSpecTest {
+
+    private static ApiClient adminClient;
+    private static OwnerDTO owner;
+    private static ConsumerDTO consumer1;
+    private static ConsumerDTO consumer2;
+    private static ConsumerDTO consumer3;
+
+    @BeforeAll
+    public static void setUp() throws ApiException {
+        adminClient = ApiClients.admin();
+        owner = adminClient.owners().createOwner(Owners.random());
+        UserDTO user = UserUtil.createUser(adminClient, owner);
+        ApiClient userClient = ApiClients.trustedUser(user.getUsername());
```

Comment:
Can we replace all `ApiClients.trustedUser(username)` with `ApiClients.basic(username, password)` through out this PR (and for all other spec tests from now on)? The ApiClients.trustedUser and ApiClients.trustedConsumer should never be used in spec tests (aside from the few tests that explicitly test those authentication methods, e.g. TrustedConsumerAuthSpecTest & TrustedUserAuthSpecTest).

Reply:
ok, done